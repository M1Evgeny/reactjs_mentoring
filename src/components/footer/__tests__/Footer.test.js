import React from "react";
import { render } from "@testing-library/react";
import { Footer } from "..";

describe("Footer test", () => {
  test("Footer snapshot", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment(<Footer />)).toMatchSnapshot();
  });
});
