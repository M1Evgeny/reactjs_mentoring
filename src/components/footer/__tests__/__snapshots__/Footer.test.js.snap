// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer test Footer snapshot 1`] = `
<DocumentFragment>
  <div
    class="footer"
  />
</DocumentFragment>
`;
